### PR TITLE
Adds a task template to the style guide

### PR DESCRIPTION
**Pull request by lamagnifica**
also testing whether the PR template gets added

### CODE REVIEW

**Comment by lamagnifica** [comment]
@catong @pdesjardins @srpearce I made this quick addition to the style guide to test my addition of the new PR template implementation.
1. The template was not added because I put my own value in the Description. The new implementation of the template will only add the boilerplate to the 1st comment aka description field for a PR. 
2. If one of you would review anyway, that would be great. I drew heavily on DITA task topics.

File: en_us/shared/manage_live_course/discussions_students.rst [hunk 1]
```diff
@@ -321,15 +327,15 @@ content-specific discussion topics.
...
+   * To add a response to the post, select **Add A Response**. When your response is complete, select **Submit**.
 
-  .. image:: ../../../shared/images/Discussion_add_response.png
-    :alt: The **Add A Response** button located between a post and its
+     .. image:: ../../../shared/images/Discussion_add_response.png
+       :alt: The **Add A Response** button located between a post and its
           responses.
+       :width: 400
+
+   * To add a comment to a response, enter text in the **Add a comment** field below the response. When your comment is complete, select **Submit**.
```
Comment (lamagnifica) [inline en_us/shared/manage_live_course/discussions_students.rst:338]: note that earlier build failures were due to the line wrapping on the four indented bullets above

**Comment by catong** [comment]
@lamagnifica Was the credentials file included in this PR by mistake?

File: en_us/edx_style_guide/source/templates/task.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.. _Task Topics:
+
+########################
+Writing Task Topics
+########################
+
+Task topics answer "How do I?" questions. They use a well-defined structure
+that describes how to accomplish a specific goal by completing a procedure. A
+task topic contains a single procedure or unit of work, which is made up of a
```
Comment (catong) [inline en_us/edx_style_guide/source/templates/task.rst]: Not clear what "unit of work" means. Omit and just go with "procedure"?

File: en_us/edx_style_guide/source/templates/task.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* A brief description. This might be added between the title and the context
+  statement.
+
+* Prerequisites. If a task has one or more prerequisites, such as use of a
+  third party tool or access with an admin password, list them before step 1 of
+  the procedure.
+
+* The result of the task, which can be useful when the result is complex, is
+  completed over time, or provides multiple possible options for proceeding
+  further.
```
Comment (catong) [inline en_us/edx_style_guide/source/templates/task.rst]: This sentence can be read in more than one way as currently worded. Suggest rewrite:
"The result of the task. Providing the expected result of a procedure is helpful to users, especially when the steps are complex, completed over time, or contain multiple options for proceeding further."

File: en_us/edx_style_guide/source/templates/task_template.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+{Optional description}
+
+Before you {complete task, list its prerequisites if any}.
+
+To {provide the context for completing the task}, follow these steps.
+
+#. {first step}.
+   {optional step result}
+
+#. {second step}.
```
Comment (catong) [inline en_us/edx_style_guide/source/templates/task_template.rst]: Perhaps add optional procedure result?

File: en_us/edx_style_guide/source/templates/concept_template.rst [hunk 5]
```diff
@@ -26,8 +29,10 @@ linked cross references to the next level headings in this file.}
 {Optional Procedure}
 ******************************
 
-Before you {complete task, list its prerequisites if any}.
+{Before you begin this task, complete this list of prerequisites if any.}
```
Comment (catong) [inline en_us/edx_style_guide/source/templates/concept_template.rst]: Strike "if any"

File: en_us/shared/manage_live_course/discussions_students.rst [hunk 6]
```diff
@@ -354,17 +360,19 @@ have already read completely.
...
+  .. image:: ../../../shared/images/Discussion_colorcoding.png
+    :alt: The discussion navigation pane with posts showing differently colored
         backgrounds and callout images.
+    :width: 200
 
 The total number of contributions in the exchange (the post and its responses
-and comments) appears in each callout image. To see the number of contributions
-that you haven't read yet, move your cursor over the callout image.
+and comments) appears in each callout image. To see how many contributions you
+still have to read, move your cursor over the callout image.
```
Comment (catong) [inline en_us/shared/manage_live_course/discussions_students.rst]: "have to" in "still have to read" is a bit ambiguous. Perhaps "to see how many contributions are still unread"?

**Comment by catong** [comment]
Not sure if the credentials and discussions_students files were included in this PR by mistake, but I did make one comment on the discussions file!

**Comment by lamagnifica** [comment]
when I went looking for examples, I noticed style guide violations in the files that I wanted to reference -- horrors! so yes, I meant to include them. thanks @catong

File: en_us/edx_style_guide/source/templates/task_template.rst [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+{Before you begin this task, complete this list of prerequisites.}
+
+To {provide the context for completing the task}, follow these steps.
+
+#. {first step}.
+   {optional step result}
+
+#. {second step}.
+
+{Optional procedure result.}
```
Comment (catong) [inline en_us/edx_style_guide/source/templates/task_template.rst:33]: Make consistent with {optional step result} in line 29 in terms of case and punctuation

**Comment by catong** [comment]
Just a tiny final nit. 👍 Thanks for adding this!

File: en_us/shared/manage_live_course/discussions_students.rst [hunk 8]
```diff
@@ -297,16 +303,16 @@ Add a Response or Comment to a Content-Specific Discussion Topic
...
+     :width: 800
 
-6. Add a response or comment.
+#. Add a response or comment.
+
+   * To add a response to the post, select **Add A Response**. When your response is complete, select **Submit**.
 
- - To add a response to the post, select **Add A Response**. When your response
-   is complete, select **Submit**.
+   * To add a comment to a response, enter text in the **Add a comment** field below the response. When your comment is complete, select **Submit**.
```
Comment (srpearce) [inline en_us/shared/manage_live_course/discussions_students.rst]: Nit: Line wrap

File: en_us/shared/manage_live_course/discussions_students.rst [hunk 9]
```diff
@@ -297,16 +303,16 @@ Add a Response or Comment to a Content-Specific Discussion Topic
...
-  .. image:: ../../../shared/images/Discussion_expand.png
-    :alt: The **Expand discussion** link under a post.
+   .. image:: ../../../shared/images/Discussion_expand.png
+     :alt: The **Expand discussion** link under a post.
+     :width: 800
 
-6. Add a response or comment.
+#. Add a response or comment.
+
+   * To add a response to the post, select **Add A Response**. When your response is complete, select **Submit**.
```
Comment (srpearce) [inline en_us/shared/manage_live_course/discussions_students.rst]: Nit: Line wrap (possibly)

**Pull request merged**